console/frontend/src/main/frontend/src/app/views/logging/logging.component.ts

```diff
@@ -84,20 +87,28 @@ export class LoggingComponent implements OnInit {
   };
 
   openDirectory(directory: string) {
-    this.loggingService.getLogging(directory).subscribe({ next: (data) => {
-      this.alert = false;
-      Object.assign(this, data);
-      this.path = data.directory;
-      if (data.count > data.list.length) {
-        this.alert = "Total number of items [" + data.count + "] exceeded maximum number, only showing first [" + (data.list.length - 1) + "] items!";
+    this.loggingService.getLogging(directory).subscribe({
+      next: (data) => {
+        this.alert = false;
+        this.originalList = data.list;
+        this.sortedlist = data.list;
+        this.directory = data.directory;
+        this.path = data.directory;
+        if (data.count > data.list.length) {
+          this.alert = "Total number of items [" + data.count + "] exceeded maximum number, only showing first [" + (data.list.length - 1) + "] items!";
```

Comment:
I prefer template strings, if the string consists of more than 2 parts. Has there been any discussion about this in the past?
```suggestion
          this.alert = `Total number of items [${data.count}] exceeded maximum number, only showing first [${data.list.length - 1}] items!`;
```

Reply:
This is basically still Niels' code so I didnt even look at it all that much, however I do agree with some minor exceptions, but your suggestion definitely looks better here